dbt_semantic_interfaces/validations/metrics.py

```diff
@@ -153,3 +154,113 @@ def validate_manifest(semantic_manifest: SemanticManifestT) -> Sequence[Validati
+
+        if metric.type_params:
+            measure = metric.type_params.measure
+            if measure is not None and measure.filter is not None:
+                try:
+                    measure.filter.call_parameter_sets
+                except Exception as e:
+                    issues.append(
+                        generate_exception_issue(
+                            what_was_being_done=f"trying to parse the measure filter on metric `{metric.name}`",
+                            e=e,
+                            context=context,
+                            extras={
+                                "traceback": "".join(traceback.format_tb(e.__traceback__)),
+                                "filter": measure.filter.where_sql_template,
+                            },
+                        )
+                    )
+
+            numerator = metric.type_params.numerator
+            if numerator is not None and numerator.filter is not None:
+                try:
+                    numerator.filter.call_parameter_sets
+                except Exception as e:
+                    issues.append(
+                        generate_exception_issue(
+                            what_was_being_done=f"trying to parse the numerator filter on metric `{metric.name}`",
+                            e=e,
+                            context=context,
+                            extras={
+                                "traceback": "".join(traceback.format_tb(e.__traceback__)),
+                                "filter": numerator.filter.where_sql_template,
+                            },
+                        )
+                    )
+
+            denominator = metric.type_params.denominator
+            if denominator is not None and denominator.filter is not None:
+                try:
+                    denominator.filter.call_parameter_sets
+                except Exception as e:
+                    issues.append(
+                        generate_exception_issue(
+                            what_was_being_done=f"trying to parse the denominator filter on metric `{metric.name}`",
+                            e=e,
+                            context=context,
+                            extras={
+                                "traceback": "".join(traceback.format_tb(e.__traceback__)),
+                                "filter": denominator.filter.where_sql_template,
+                            },
+                        )
+                    )
+
+            for derived_metric in metric.type_params.metrics or []:
+                if derived_metric.filter is not None:
+                    try:
+                        derived_metric.filter.call_parameter_sets
+                    except Exception as e:
+                        issues.append(
+                            generate_exception_issue(
+                                what_was_being_done=f"trying to parse derived metric filter on metric `{metric.name}`",
```

Comment:
This is confusing for the user, because it's the input metric, not the derived metric:

```suggestion
                                what_was_being_done=f"trying to parse filter for input metric `{derived_metric.name}` on metric `{metric.name}`",
```